tasks/main.yaml

```diff
@@ -1,10 +1,12 @@
 ---
-
-- name: Test if distribution is RedHat
+- name: Distribution test
   assert:
     that: >
-      ansible_os_family == "RedHat" and
-      ansible_distribution_major_version | int >= 6
+      (
+        ansible_os_family == "RedHat" and
```

Comment:
Please could you replace the double quotes by single quotes here and in all `when` clauses bellow?